insights/parsers/systemctl_show.py

```diff
@@ -29,6 +33,8 @@ class SystemctlShowServiceAll(CommandParser, dict):
         LimitNOFILE=65536
         LimitMEMLOCK=
         LimitLOCKS=18446744073709551615
+        DummyKey1 = Value1
+        DummyKey1 = Value2
```

Comment:
Is this valid for there to be two keys with the same name for a particular services in the output of this command?